nav2_bringup/bringup/launch/nav2_multi_tb3_simulation_launch.py

```diff
@@ -132,6 +139,8 @@ def generate_launch_description():
             source_file=rviz_config_file,
             replacements={'<robot_namespace>': ('/' + robot['name'])})
 
+        params_file = eval(robot['name'] + '_params_file')
```

Comment:
Can we change this to:

```
params_file = LaunchConfiguration(robot['name'] + '_params_file')
```

Then we can get rid of the LaunchConfiguration lines above.

Maybe it's just me, but I find the use of eval here difficult to read.

Reply:
So on first glance I agree with you, but something that came to mind was that since we're doing this `eval` in a loop, that if someone changes 2 robots to `N` robots and there aren't `N` parameter files, you're not going to be able to statically look at the initialization of the launch configurations and see that there are things missing. 

If you don't think that's a real problem, I can do that -- I'm just thinking and if there's a bug, some analyzer isn't going to catch that.